docker-compose/go-app-ussd-registration.zul_ZA.json

```diff
@@ -0,0 +1,725 @@
+{
+    "": {
+        "pot-creation-date": "2020-10-20 12:10:+0000",
+        "project-id-version": "",
+        "language-team": "zulu",
+        "language": "",
+        "mime-version": "1.0",
+        "content-type": "text/plain; charset=utf-8",
+        "content-transfer-encoding": "8bit",
+        "po-revision-date": "YEAR-MO-DA HO:MI+ZONE",
+        "last-translator": "Chima"
+    },
+    "WhatsApp": [
+        null,
+        "Whatsapp"
```

Comment:
I don't think that we should be having these files copied into the docker compose setup. Rather, the `config/....json` files should be copied into the docker image, and then the docker compose setup should read from those files. That way, we don't have duplicate files where we can forget to update one, and the docker compose setup is closer to what it is in production